### DISCUSSION TITLE
Apollo client graphql subscription

### DISCUSSION THREAD

**Discussion by ghosteye1** [quasarframework/quasar#8920] [Quasar App Extensions] [answered]
I'm trying to integrate a subscription to my quasar app. I already added the @quasar/apollo and I can't find where I could set up the WebSocket URL for a subscription. Plz help.

**Comment by smolinari** [comment]
Just like http link, you create a ws link. Check out this reply (this is assuming you are using v1 of the Apollo AE):

https://github.com/quasarframework/app-extension-apollo/issues/37#issuecomment-619650193

Scott

**Reply by ghosteye1** [reply]
Thanks for the reply. I already tried abv but didn't work due to the missing header. This lib solved my issue
https://github.com/absinthe-graphql/absinthe-socket/tree/master/packages/socket-apollo-link

**Comment by ghosteye1** [comment, marked as answer]
My Backend is elixir based phenix application. Following lib resolved my issue. 
https://github.com/absinthe-graphql/absinthe-socket/tree/master/packages/socket-apollo-link
